src/locales/langs/en.ts

```diff
@@ -157,6 +157,65 @@ const en = {
       invalid_code: "Entered code doesn't match the code which was sent to your email.",
     },
   },
+  profile: {
+    pageTitle: 'Profile',
+    headerTitle: 'Complete Profile',
+
+    qualification: 'Qualification',
+    workExperience: 'Work Experience',
+    services: 'Services',
+    availability: 'Availability',
+    rates: 'Rates',
+    bio: 'Bio',
+  },
+  profileQualification: {
+    certificationNameMaxLength: 'Certification name must be at most 100 characters',
```

Comment:
it would be better to get the max amount of characters from constansts

Reply:
done